config.json

```diff
@@ -706,7 +706,9 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "strings",
+        "list",
```

Comment:
Could you please change it to `lists` to be consistent with https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt